Makefile

```diff
@@ -138,23 +138,26 @@ get-cluster-credentials: read-cluster-config set-azure-account ## make <config>
 	az aks get-credentials --overwrite-existing -g ${RESOURCE_NAME_PREFIX}-tsc-${CLUSTER_SHORT}-rg -n ${RESOURCE_NAME_PREFIX}-tsc-${CLUSTER}-aks
 	kubelogin convert-kubeconfig -l $(if ${GITHUB_ACTIONS},spn,azurecli)
 
-aks-console: get-cluster-credentials
```

Comment:
Will any of these changes affect the infra team?
I see in Apply we alias the commands https://github.com/DFE-Digital/apply-for-teacher-training/blob/0d9f56abc3c0bbe22d99fc904b955d0bfa466609/Makefile#L133

Reply:
Good question. I added @RMcVelia. Can you review this PR please?

Reply:
If you change (instead of alias) then there are other places that the code will need to be changed. e.g. github workflows are hardcoded to env_aks

Reply:
thank you @RMcVelia. I will alias everything